src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PAY_PER_DAY = 40;
+  const SUM_PER_DAY = days * PAY_PER_DAY;
```

Comment:
It is better not to use a constant here: Constants are for values that remain unchanged throughout the execution of the program. Because SUM_PER_DAY depends on other variables, its value can vary depending on the context.